modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
Well, you'll want the module to still take in path and prefix, since it's the module itself that'd get passed to convert.inputs. But that module can then call the model-specific veg2model.[MODEL]. As you note earlier, no point in having it write to file in between.